CMakeLists.txt

```diff
@@ -122,11 +123,13 @@ if(FCL_WITH_OCTOMAP)
           set(OCTOMAP_LIBRARIES "octomap;octomath")
       endif()
   endif()
-  if (OCTOMAP_FOUND OR (OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS))
-    string(REPLACE "." ";" VERSION_LIST ${OCTOMAP_VERSION})
-    list(GET VERSION_LIST 0 OCTOMAP_MAJOR_VERSION)
-    list(GET VERSION_LIST 1 OCTOMAP_MINOR_VERSION)
-    list(GET VERSION_LIST 2 OCTOMAP_PATCH_VERSION)
+  if(OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS AND OCTOMAP_VERSION)
+    if(NOT OCTOMAP_MAJOR_VERSION AND NOT OCTOMAP_MINOR_VERSION AND NOT OCTOMAP_PATCH_VERSION)
```

Comment:
The current second (non-pkg-config) way  of finding octomap does not define OCTOMAP_VERSION. What do you currently do there?